src/app/utils/userLocationUtils.ts

```diff
@@ -0,0 +1,13 @@
+const LOCATION_API = process.env.REACT_APP_LOCATION_API_URL;
+
+export const getUserLocation = async (): Promise<{ ip: string; location: string }> => {
+  const response = await fetch(`${LOCATION_API}`, {
```

Comment:
use axios instead of fetch. Why didn't have added this call to sdk as other ones?

Reply:
Fetch adds the auth headers automatically and we don't need them to call this endpoint. 

Regarding the SDK, I can add it, yes.